build/aca-machine-init.sh

```diff
@@ -36,7 +36,7 @@ echo "2--- installing librdkafka ---" && \
 
 echo "3--- installing cppkafka ---" && \
     apt-get update -y && apt-get install -y cmake 
-    git clone https://github.com/mfontanini/cppkafka.git /var/local/git/cppkafka && \
+    git clone https://hub.fastgit.org/mfontanini/cppkafka.git /var/local/git/cppkafka && \
```

Comment:
What is the reason to use hub.fastgit.org mirror? If this is needed for your access, maybe we can pass in that option is a command line parameter.